src/test/java/core/basesyntax/FruitStorageServiceTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.fruitservice.FruitStorage;
+import core.basesyntax.fruitservice.FruitStorageService;
+import core.basesyntax.fruitservice.Transaction;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitStorageServiceTest {
+    private Transaction transaction = Transaction.build("b", "banana", "200", "2010-11-20");
+    private FruitStorageService fruitStorageService = new FruitStorageService();
+    private FruitStorage fruitStorage = new FruitStorage();
+
+    @Test
+    public void addTest() {
+        List<Transaction> testList = new ArrayList<>();
+        testList.add(transaction);
+        fruitStorageService.addToStorage(transaction);
+        Assert.assertEquals(testList, fruitStorage.getStorage());
+    }
+
+    @After
+    public void ClearAfterAddition() {
```

Comment:
давай перенесемо цей метод вкінець класу 
```suggestion
    public void clearAfterAddition() {
```